### PR TITLE
Bump release version number to 2.5

### PR DESCRIPTION
**Pull request by daschuer**
This a version bump PR similar to #400 
But I must admit I don't know what I am doing. 
Is the traget branch OK, or do we need to adjust all branches?

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -102,9 +102,9 @@
...
 smv_branch_whitelist = r"^(main|(\d+)\.(\d+))$"
 smv_remote_whitelist = r"^origin$"
-# Mark all branches in the form "N.M" as released except "2.5"
-smv_released_pattern = r"^.*/(?!2\.5)(\d+)\.(\d+)$"
-smv_latest_version = r"2.4"
+# Mark all branches in the form "N.M" as released except "2.6"
+smv_released_pattern = r"^.*/(?!2\.6)(\d+)\.(\d+)$"
+smv_latest_version = r"2.5"
 smv_outputdir_format = "{config.version}/{config.language}"
 notfound_urls_prefix = "/2.4/en/"
```
Comment (Holzhaus) [inline source/conf.py:109]:
```suggestion
notfound_urls_prefix = "/2.5/en/"
```

**Comment by Holzhaus** [comment]
Pre-commit for `.tx/config` fails btw

**Comment by daschuer** [comment]
Done

**Comment by Holzhaus** [comment]
thanks

**Pull request merged**